sos/report/plugins/ceph_common.py

```diff
@@ -95,12 +95,22 @@ def setup(self):
                     "/var/snap/microceph/common/logs/ceph.audit.log*",
                 ])
 
-            self.add_cmd_output("microceph status", subdir="microceph")
             self.add_cmd_output("snap info microceph", subdir="microceph")
-            # Collect all but exclude any keyrings
-            self.add_cmd_output("microceph cluster sql 'select * from config \
-                                 where key NOT LIKE \"%keyring%\"'",
-                                subdir="microceph")
+
+            cmds = [
+                'client config list',
+                'cluster config list',
```

Comment:
@pponnuvel is there any private information on config list that should be censored?

Reply:
@dnegreira  This gets the configuration settings for daemons. This is sort of the equivalent of "ceph config show" in the standard Ceph (which we already collect for Ceph). This currently supports just two: `cluster_network`, and `osd_pool_default_crush_rule`. And don't expect any confidential here in the future either.